doc-examples/jdbc-example-java/src/main/java/example/BookRepository.java

```diff
@@ -109,6 +109,11 @@ interface BookRepository extends CrudRepository<Book, Long> { // <2>
     Long calculateSum(@NonNull Long bookId);
     // end::procedure[]
 
+    // tag::updateReturning[]
+    @Procedure
+    Long calculateSum(@NonNull Long bookId);
+    // end::updateReturning[]
+
```

Comment:
This is failing 
```
/home/runner/work/micronaut-data/micronaut-data/doc-examples/jdbc-example-java/src/main/java/example/BookRepository.java:114: error: method calculateSum(@io.micronaut.core.annotation.NonNull Long) is already defined in interface BookRepository

    Long calculateSum(@NonNull Long bookId);
```
because there is already `calculateSum` with `// tag::procedure[]`

Reply:
Committed by mistake